src/PerfView/CommandLineArgs.cs

```diff
@@ -421,8 +420,9 @@ private void SetupCommandLine(CommandLineParser parser)
                     KernelTraceEventParser.Keywords.Interrupt;
             }
 
+            bool Finalizers = false;
             parser.DefineOptionalQualifier("DumpHeap", ref DumpHeap, "Capture a heap snapshot on profile stop");
-            parser.DefineOptionalQualifier("Finalizers", ref Finalizers, "Track the number of objects of each type finalized.");
+            parser.DefineOptionalQualifier("Finalizers", ref Finalizers, "(Obsolete) This option is accepted for backward compatibility with tooling, but is no longer used.");
```

Comment:
Please simply remove this option (don't bother with a no-op obsolete qualifier).    We don't guarantee compatibility of command line options.  The idea is that PerfView's xcopy deployment allows any scripts that have wired in particular command line options to also carry the particular version of PerfView as well.